css/_variables.scss

```diff
@@ -89,7 +89,7 @@ $sidebarWidth: 200px;
  */
 $tooltipsZ: 901;
 $toolbarZ: 900;
-$overlayZ: 800;
```

Comment:
So the problem is that we want to show the overlays which shows some message on top of the toolbar, but we want to show it below for the call overlay. Will have to split that